ui/widgets/nuxeo-user-avatar.js

```diff
@@ -309,9 +281,9 @@ import '../nuxeo-icons.js';
           ];
 
           const alphabetPosition = alphabet.indexOf(name.charAt(0).toLowerCase());
+          const hue = this.__generateHue();
```

Comment:
Is this variable just for readability purposes, correct?

Reply:
Indeed. Shall I remove it?

Reply:
Personally, I don't see a need for the variable. But I also don't have strong feelings about keeping it.
Let's wait for other review ;)